src/shapes/Object/InteractiveObject.ts

```diff
@@ -491,8 +467,7 @@ export class InteractiveFabricObject<
     this.setCoords();
     this.forEachControl((control, key) => {
       if (control.getVisibility(this, key)) {
-        const p = this.oCoords[key];
-        control.render(ctx, p.x, p.y, options, this);
+        control.render(ctx, this.oCoords[key], options, this);
```

Comment:
Can't you call `control.renderConnection` before `control.render` to avoid the breaking change?

Reply:
I thought of that...
I don't like it because then the rendering call is not centralized so overriding becomes annoying and hard to customize.
I suggest exposing a calling method, maybe named `renderControl` that will call `renderConnection` and `render` and eventually deprecate `render` since I have a few suggestions in terms of matrices to pass to this method, now that I reworked coords.
Thoughts?

Reply:
There is nothing to customize in renderConnection right now.
To me it seems totally fine to do those things slowly, this change isn't fixing anything, is moving the line responsibility from canvas to the control. And since we dont want to break this interface for now is basically the only thing we can do.

Regarding:
- i will have a method that call the old render + renderConnection
VS
- i will call render + renderConnection from here

little changes since the point is that we are not killing the old render, and so your freedom is somehow limited.
The only thing you would gain is that you can pass more things to a new render, and use them in renderConnection.

Reply:
great

Reply:
I will use it for my fix of coords as well